src/epiviz.gl/webgl-vis.js

```diff
@@ -54,6 +56,17 @@ class WebGLVis {
     this.sendDrawerState(this.mouseReader.getViewport());
   }
 
+  /**
+   * Set the flag for uniDirectionSelectEnabled to be used by the mouse reader
+   * to determine whether user is allowed to do horizontal or vertical selection
+   * @param {Boolean} enabled whether or not to enable uniDirectionSelect
+   * @memberof WebGLVis
+   * @returns {void}
+   */
+  setUniDirectionSelectEnabled(enabled) {
+    this.uniDirectionSelectionEnabled = enabled;
+  }
+
```

Comment:
shouldn't this be in-sync with `this.POSSIBLE_MOUSE_READER_OPTIONS` ?

Reply:
or whatever variable controls the state

Reply:
Wasn't aware of this, Thanks for letting me know.

Updated.